supabase/migrations/06_connectors.sql

```diff
@@ -1,14 +1,37 @@
+create domain jsonb_internationalized_value as jsonb check (
+  (value is null) OR -- This feels wrong, but without it the check constraint fails on nulls
+  (jsonb_typeof(value) = 'object' AND 
+  (value->'en-US' IS NOT NULL))
+);
+comment on domain jsonb_internationalized_value is
+  'jsonb_internationalized_value is JSONB object which is required to at least have en-US internationalized values';
+
+CREATE OR REPLACE FUNCTION 
+generate_opengraph_value( opengraph_raw jsonb, opengraph_patch jsonb, field text )
+RETURNS jsonb_internationalized_value
+AS $CODE$
+BEGIN
+    RETURN json_build_object('en-US',internal.jsonb_merge_patch(opengraph_raw, opengraph_patch) #>> ('{"en-US", "'|| field ||'"}')::text[]);
+END
+$CODE$
+LANGUAGE plpgsql IMMUTABLE;
 
 -- Known connectors.
 create table connectors (
   like internal._model including all,
 
   external_url           text not null,
   image_name             text unique not null,
+  -- To be deleted in a future change --
   open_graph             jsonb_obj
     generated always as (internal.jsonb_merge_patch(open_graph_raw, open_graph_patch)) stored,
   open_graph_raw         jsonb_obj,
   open_graph_patch       jsonb_obj,
+  --        End to be deleted        --
+  title                  jsonb_internationalized_value generated always as (generate_opengraph_value(open_graph_raw, open_graph_patch,'title')) stored,
+  short_description      jsonb_internationalized_value generated always as (generate_opengraph_value(open_graph_raw, open_graph_patch,'description')) stored,
+  logo_url               jsonb_internationalized_value generated always as (generate_opengraph_value(open_graph_raw, open_graph_patch,'image')) stored,
+  recommended            boolean not null generated always as (case when internal.jsonb_merge_patch(open_graph_raw, open_graph_patch)->'en-US'->>'recommended'::text = 'True' then TRUE else FALSE end) stored,
```

Comment:
Should this be `'true'`, rather than `'True'` ?

Reply:
You'd think so, wouldn't you

<img width="315" alt="Screen Shot 2022-09-06 at 13 35 02" src="https://user-images.githubusercontent.com/4368270/188702268-4777af54-b86c-4b82-955e-7ae710d2f413.png">

Reply:
... umm, okay then. 🤣